src/main/java/com/snowflake/kafka/connector/internal/EnableLogging.java

```diff
@@ -1,63 +0,0 @@
-/*
- * Copyright (c) 2022 Snowflake Inc. All rights reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the
- * "License"); you may not use this file except in compliance
- * with the License.  You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied.  See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-
-package com.snowflake.kafka.connector.internal;
-
-// The purpose of this class is for non-static inheritors to easily log information
-public abstract class EnableLogging {
```

Comment:
IIRC, this was added by Revi, could you check with her to see why we decide to go this way?

Reply:
I didn't add this class, just added to it as it was there already. I also am not sure why we went this way, so fully support removing it

Reply:
Ack